deepchem/__init__.py

```diff
@@ -17,3 +17,4 @@
 import deepchem.dock
 import deepchem.molnet
 import deepchem.rl
+import deepchem.deep_dft
```

Comment:
Why is this a top level module? This should not be a top level module @GreatRSingh . This needs to be moved to a sub module of dc.models